src/core/FileManagerServices.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * FileManagerServices.cpp - Helper file for cross platform file management
+ *
+ * Copyright (c) 2025 Andrew Wiltshire
```

Comment:
Mostly optional, but you might want to put your email here (and in other places where you have copyrighted the source code). Don't feel pressured to though, its mostly optional as mentioned. @tresf might have more to say about this.